src/main/java/frc/robot/commands/Auto/ScoreHighCubeBalance.java

```diff
@@ -36,18 +37,18 @@ public ScoreHighCubeBalance(DriveSubsystem m_drive, FieldSim m_fieldSim, ArmInOu
         // run instantly
         new InstantCommand(m_LED::autoStart),
 
-        new ManipulatorMidGoal(m_ArmPivot, m_ArmInOut, m_Wrist, m_LED).withTimeout(2),
-        new RunCommand(m_Gripper::GripperFast, m_Gripper).withTimeout(2),
-        new RunCommand(m_Gripper::GripperStop, m_Gripper).withTimeout(1),
+        new ManipulatorMidGoal(m_ArmPivot, m_ArmInOut, m_Wrist, m_LED).withTimeout(1),
+        new RunCommand(m_Gripper::GripperFast, m_Gripper).withTimeout(1),
+        new RunCommand(m_Gripper::GripperStop, m_Gripper).withTimeout(0.1),
         // new FollowTrajectory(m_drive, m_fieldSim, "Drive Backward", m_ArmInOut,
         // m_ArmPivot, m_Gripper, m_Wrist, m_LED),
-        new ManipulatorZero(m_ArmPivot, m_ArmInOut, m_Wrist, m_LED).withTimeout(2),
+        new ManipulatorZero(m_ArmPivot, m_ArmInOut, m_Wrist, m_LED).withTimeout(0.1),
         new RunCommand(() -> m_drive.drive(0.1, 0, 0, true, true), m_drive).withTimeout(0.1),
-        new ManipulatorGround(m_ArmPivot, m_ArmInOut, m_Wrist, m_LED).withTimeout(2),
-        new RunCommand(m_Wrist::WristPositionManualUp, m_Wrist).withTimeout(1),
-        new RunCommand(() -> m_drive.drive(0.1, 0, 0, true, true), m_drive).withTimeout(5),
-        new BalanceChargeStation(m_drive, false).withTimeout(5),
-        // new ManipulatorZero(m_ArmPivot, m_ArmInOut, m_Wrist, m_LED).withTimeout(2)2
-        new InstantCommand(m_LED::autoEnd));
+        new ManipulatorGroundAuto(m_ArmPivot, m_ArmInOut, m_Wrist, m_LED).withTimeout(0.1),
+        new RunCommand(() -> m_drive.drive(0.1, 0, 0, true, true), m_drive).withTimeout(4.7),
+        new InstantCommand(m_LED::autoEnd),
```

Comment:
For all autos:

It might be worth moving `m_LED::autoStart` and `m_LED::autoEnd` out of each command & into `RobotContainer::autoInit` and `RobotContainer::teleopInit` so each command doesn't need to make sure it's setting it